src/components/custom/other-platforms.js

```diff
@@ -207,6 +231,7 @@ export const OtherPlatform = ({ header, subHeader, exclude, is_nav, is_ppc_redir
             {exclude.toLowerCase() !== 'dtrader' && <TraderCard />}
             {exclude.toLowerCase() !== 'dbot' && <BotCard />}
             {exclude.toLowerCase() !== 'dmt5' && <DMT5Card is_ppc_redirect={is_ppc_redirect} />}
+            {exclude.toLowerCase() !== 'derivx' && <DerivXCard is_ppc_redirect={is_ppc_redirect} />}
```

Comment:
```suggestion
            {exclude.toLowerCase() !== 'derivx' && <DerivXCard />}
```

Reply:
I don't think you will need the `is_ppc_redirect` prop here as the landing page will not be nested under `/landing/*`.